thesslstore_plugin.php

```diff
@@ -371,5 +379,4 @@ private function serviceFieldsToObject(array $fields) {
 
         return $data;
     }
-}
-?>
+}
```

Comment:
It's good to remove the unnecessary closing php tag, but there should exist a blank new line at the end of the file.

Reply:
good catch